lib/model/dateOfBirth.dart

```diff
@@ -0,0 +1,16 @@
+class DateOfBirth {
```

Comment:
the file of this class should follow the convention: underscore_lower_case like in time_zone and contact_data, when the name has more than one word

Reply:
got it